{{cookiecutter.repository_name}}/setup.py

```diff
@@ -11,22 +11,21 @@
 with open('HISTORY.md') as history_file:
     history = history_file.read()
 
-requirements = [
+install_requires = [
     {%- if cookiecutter.command_line_interface|lower == 'click' %}'Click>=6.0',{%- endif %}
     {%- if cookiecutter.support_py2 == 'y' %}'six',{%- endif %}
 ]
 
-setup_requirements = [
+setup_requires = [
     'pytest-runner>=2.11.1',
 ]
 
-test_requirements = [
-    'coverage>=4.5.1',
+tests_require = [
     'pytest>=3.4.2',
-    'tox>=2.9.1',
+    'pytest-cov>=2.6.0',
```

Comment:
why is this required under tests_require? should it go in development_requires?

Reply:
It is needed here, because the `make test` command now includes de `--cov` part, and the `tox` config installs `dev` extras only for docs and lint. The default test setup installs only the `test` requirements.

Reply:
ok